aten/src/ATen/native/cuda/Reduce.cuh

```diff
@@ -704,29 +726,29 @@ struct ReduceOp {
   }
 
   template<class T>
-  C10_DEVICE void set_results(const T x, const index_t base_offset) const {
+  C10_DEVICE void set_results(const T x, const at::detail::Array<index_t, num_outputs + 1> base_offset) const {
     assert(noutputs == 1);
-    auto res = (out_scalar_t*)((char*)dst[0] + base_offset);
+    auto res = (out_scalar_t*)((char*)dst[0] + base_offset[0]);
     *res = x;
   }
 
   //Currently implemented for max of two outputs
   template<class T1, class T2>
-  C10_DEVICE void set_results(const thrust::pair<T1, T2> x, const index_t base_offset) const {
+  C10_DEVICE void set_results(const thrust::pair<T1, T2> x, const at::detail::Array<index_t, num_outputs + 1> base_offset) const {
     if (noutputs >= 1) {
-      auto res0 = (T1*)((char*)dst[0] + base_offset);
+      auto res0 = (T1*)((char*)dst[0] + base_offset[0]);
       *res0 = x.first;
     }
     if (noutputs >= 2) {
       // base offset is computed assuming element size being sizeof(T1), so we need to make a
```

Comment:
is this comment out of date now?

Reply:
I guess so because TensorIterator gives you the stride data in terms of bytes.

Reply:
You are right, it is out of date now. Deleted from the code.